global_planner/src/library/cell.cpp

```diff
@@ -70,14 +70,17 @@ Cell Cell::getNeighborFromYaw(double yaw) const {
 }
 
 // Returns the neighbors of the Cell whose risk influences the Cell
-std::vector<Cell> Cell::getFlowNeighbors() const {
-  return std::vector<Cell>{
-      Cell(std::tuple<int, int, int>(xIndex() + 1, yIndex(), zIndex())),
-      Cell(std::tuple<int, int, int>(xIndex() - 1, yIndex(), zIndex())),
-      Cell(std::tuple<int, int, int>(xIndex(), yIndex() + 1, zIndex())),
-      Cell(std::tuple<int, int, int>(xIndex(), yIndex() - 1, zIndex())),
-      Cell(std::tuple<int, int, int>(xIndex(), yIndex(), zIndex() + 1)),
-      Cell(std::tuple<int, int, int>(xIndex(), yIndex(), zIndex() - 1))};
+std::vector<Cell> Cell::getFlowNeighbors(int radius) const {
+  std::vector<Cell> cells;
+  for (int x = -radius; x < radius; x++) {
+    for (int y = -radius; x < radius; x++) {
+      for (int z = -radius; x < radius; x++) {
```

Comment:
```suggestion
      for (int z = -radius; z <= radius; z++) {
```
Same here, only checking & increasing x, not z